src/toil/test/src/importExportFileTest.py

```diff
@@ -62,7 +62,7 @@ def _importExportFile(self, options, fail):
 
     def _importExport(self, restart):
         options = Job.Runner.getDefaultOptions(self._getTestJobStorePath())
-        options.logLevel = "INFO"
+        options.logLevel = "DEBUG"
```

Comment:
Same comment as previous. For all of the options, I think they should just use the default.

Reply:
I like leaving it in, but `info` or `critical` will reduce the logs it produces.  Don't set this to `debug`.